src/components/Note.jsx

```diff
@@ -7,7 +7,7 @@ function Note(props) {
   }
 
   return (
-    <div className='note' key={props.key} id={props.id}>
+    <div className='note' id={props.id}>
```

Comment:
I have a feeling you may confuse the `key` attribute and the `id` attribute of the `div` tag. The `id` attribute has a special meaning in HTML, and I'd strongly advise not to use it in this case. https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id